### ISSUE TITLE
Item rune sockets

### ISSUE THREAD

**Issue by Kvan7** [Kvan7/Exiled-Exchange-2#176]
Add support for item's rune sockets, basically using the empty sockets filter in the weapon section. 

- [x] #177
- [x] #178
- [x] #283
- [ ] #208 
- [x] #282
- [x] #292

**Comment by Kvan7** [comment]
Realizing this is quite a bit more complicated so added a milestone

**Comment by judgegallows** [comment]
I just updated to 0.1.5 and searching for sceptres/wands/staff doesn't work properly now due to the empty rune socket filter. The search filters for a min search of 1 socket and when you uncheck the option it searches for 0 min sockets which breaks the search. The trade site needs the field to be blank to work for these weapons.

**Comment by Kvan7** [comment]
oh those just don't have sockets. interesting. hotfix incoming yay

**Comment by Kvan7** [comment]
imagine playing magic lol :)

**Comment by judgegallows** [comment]
Thanks! Also just noticed that focuses can have a socket but it isn't supported.

**Comment by Kvan7** [comment]
yeah i saw that after I had started the release build, not breaking stuff so i'll just pick it up tomorrow

**Comment by judgegallows** [comment]
yeah no worries. you work quick. Thanks again!
